### DISCUSSION TITLE
Skip a migration for specific database url  (dev/prod)

### DISCUSSION THREAD

**Discussion by ajaishankar** [amacneil/dbmate#454] [Support]
Hi

Is there a way to not run a migration based on database url?

For example I have a migration file to seed some (test) data in the development database.

But I do not want to run this migration file on production.

Is there a some strategy we could use to enable this?

Thanks!

**Comment by gregwebs** [comment]
If it loads against the existing schema, you can use any tooling for the job, including putting it into a separate folder and using dbmate against that folder.

If the data needs to be loaded at a particular point of a past schema then this feature doesn't exist in dbmate. There is a PR for templating variables and with that feature it might be possible to write an if statement into the migration.

**Comment by dossy** [comment]
@ajaishankar Take a look at #424, which was implemented in #428.  Does that address your need?

**Comment by ajaishankar** [comment]
@dossy 

The approach I had followed was what @gregwebs suggested.

Put seed data in a separate `./db/dev-migrations` directory and run that only in dev/test env `dbmate -d ./db/dev-migrations`

I really did not really need merging since ours was a pretty simple app (schema in one directory, dev seed data in another)

Thanks everyone for such a nice and simple migration tool!

Ajai
